css/src/tokens/themes.scss

```diff
@@ -31,7 +31,8 @@ $themes: (
 		'hover-invert': $palette-grey-10,
 		'code-header': $palette-grey-30,
 		'code-block': $palette-grey-20,
-		'control-border': $palette-grey-80,
```

Comment:
changing that value also affected notification component. Can you add the `notification-border` with these values and apply it to the component, please?

Before:

![image](https://github.com/user-attachments/assets/6aa3365e-e894-44cd-954f-22b1243f3954)


In this PR:

![image](https://github.com/user-attachments/assets/4d9681cc-7615-4f66-ab73-20e6d3b2839a)

Reply:
Oh, it also affected the checkbox and I think it's not passing the accessibility requirements. Hold on on changes, I need to double check.